guides/common/modules/proc_using-vmware-cloud-init-and-userdata-templates-for-provisioning.adoc

```diff
@@ -26,18 +26,17 @@ When you set up the compute resource and images for VMware provisioning in {Proj
 * VMware vCenter applies customization for the virtual machine's identity, including the host name, IP, and DNS
 * The virtual machine builds, `cloud-init` is invoked and calls back {Project} on port `80`, which then redirects to `443`
 
-.Port and Firewall Requirements
+.Prerequisites
+* Ensure that you configure port and firewall settings to open any necessary connections.
```

Comment:
```suggestion
* Configure port and firewall settings to open any necessary connections.
```

Reply:
This, or maybe past tense because it should have been done before starting this procedure?

Looking at the current docs, there seems to be no consistent wording: `rg -A 3 "\.Prerequisite"`. (better idea: "find | xargs sed /pattern/ until first empty line" but I couldn't find that right now; will look it up if someone is interested)

Reply:
> This, or maybe past tense because it should have been done before starting this procedure?

The Red Hat supplementary guide states that it's okay to use passive voice for prerequisites in cases like these: https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites  "Port and firewall settings are configured", or a variant thereof, might actually be the most appropriate then.
> Looking at the current docs, there seems to be no consistent wording: `rg -A 3 "\.Prerequisite"`. (better idea: "find | xargs sed /pattern/ until first empty line" but I couldn't find that right now; will look it up if someone is interested)

As for consistency in wording when it comes to prerequisites, I wouldn't worry about that too much. Sometimes it's the current user who is supposed to perform the prereq (_Configure XY_), sometimes it might not be the current user (_XY is configured_). (My two cents.)